### DISCUSSION TITLE
Used by counter not showing main project

### DISCUSSION THREAD

**Discussion by jperl** [community/community#22960] [New to GitHub]
<p>We want the used by counter to be for the <code>qawolf</code> package but it is the only package missing from the settings “Used by counter” dropdown. I tried to submit a support ticket but have not heard back about it. Thanks!</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/e7590cbeae8109bba5ac173ad8c7ce28dc68f4f1.png" data-download-href="/uploads/short-url/x0B5cJmOsytzWC1Z4UhZtcDyvfP.png?dl=1" title="Screen Shot 2019-11-10 at 5.55.06 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/1136652/181074058-48130bf1-acc0-4d58-a34c-24401b874585.png" alt="Screen Shot 2019-11-10 at 5.55.06 PM.png" data-base62-sha1="x0B5cJmOsytzWC1Z4UhZtcDyvfP" width="322" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/e7590cbeae8109bba5ac173ad8c7ce28dc68f4f1_2_322x500.png, https://aws1.discourse-cdn.com/github/optimized/1X/e7590cbeae8109bba5ac173ad8c7ce28dc68f4f1_2_483x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/e7590cbeae8109bba5ac173ad8c7ce28dc68f4f1_2_644x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/e7590cbeae8109bba5ac173ad8c7ce28dc68f4f1_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-11-10 at 5.55.06 PM.png</span><span class="informations">708×1096 85.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/47217e0675f8f816c27323541a9a0010669a0da5.png" data-download-href="/uploads/short-url/a9fEKDxdehZPk3sdeK9aYzFjgMJ.png?dl=1" title="Screen Shot 2019-11-10 at 5.55.19 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/1136652/181074059-f74e1192-3ef0-421f-abae-9a983a490431.png" alt="Screen Shot 2019-11-10 at 5.55.19 PM.png" data-base62-sha1="a9fEKDxdehZPk3sdeK9aYzFjgMJ" width="317" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/47217e0675f8f816c27323541a9a0010669a0da5_2_317x500.png, https://aws1.discourse-cdn.com/github/optimized/1X/47217e0675f8f816c27323541a9a0010669a0da5_2_475x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/47217e0675f8f816c27323541a9a0010669a0da5_2_634x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/47217e0675f8f816c27323541a9a0010669a0da5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-11-10 at 5.55.19 PM.png</span><span class="informations">694×1092 85.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p><a href="https://github.com/qawolf/qawolf/blob/master/packages/qawolf/package.json" rel="nofollow noopener">https://github.com/qawolf/qawolf/blob/master/packages/qawolf/package.json</a></p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/jperl">@jperl</a>,</p>
<p>Thank you for being here and our apologies for the delayed response. There was a limit to the number of packages visible in the used by dropdown, but I believe this has since been removed.</p>
<p>I’m hoping you’re able to select the missing package now?</p>

**Comment by jperl** [comment]
<p>It is fixed, thanks! Now I just need to get that counter to show more repos :stuck_out_tongue:</p>
